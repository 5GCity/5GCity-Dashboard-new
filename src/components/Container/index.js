/**
 * Container Component
 * Please write a description
 *
 * @author Your Name <youremail@ubiwhere.com>
 */
import React from 'react'
import styled from 'styled-components'

export default ({ children, ...props }) => (
  <Wrapper {...props}>{children}</Wrapper>
)

const Wrapper = styled.div`
  max-width: 1200px;
  margin: 0 auto;
`
