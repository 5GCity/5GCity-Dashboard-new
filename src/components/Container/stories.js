/**
 * Container Component Stories
 * Please write a description or remove this line
 *
 * @author Your Name <youremail@ubiwhere.com>
 */
import React from 'react'
import { storiesOf } from '@storybook/react'

import Container from './index'

const State1 = () => (
  <Container />
)

storiesOf('Container', module)
  .add('primary', State1)
