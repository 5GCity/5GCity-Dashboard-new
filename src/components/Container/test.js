/**
 * Brand Component Tests
 *
 * @author Hugo Fonseca <hfonseca@ubiwhere.com>
 */
import React from 'react'
import renderer from 'react-test-renderer'
import Container from './index'
import 'jest-styled-components'

it('renders as it should', () => {
  const tree = renderer
    .create(
      <Container>
        <div>children element</div>
      </Container>
    )
    .toJSON()
  expect(tree).toMatchSnapshot()

  expect(tree).toHaveStyleRule('max-width', '1200px')
  expect(tree).toHaveStyleRule('margin', '0 auto')
})
