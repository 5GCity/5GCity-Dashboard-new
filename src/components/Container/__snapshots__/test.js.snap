// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders as it should 1`] = `
.c0 {
  max-width: 1200px;
  margin: 0 auto;
}

<div
  className="c0"
>
  <div>
    children element
  </div>
</div>
`;
